api/tests/unit/application/organizations/organization-controller_test.js

```diff
@@ -622,5 +631,77 @@ describe('Unit | Application | Organizations | organization-controller', () => {
     });
 
   });
-})
-;
+
+  describe('#getCampaigns', () => {
+
+    let sandbox;
+
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create();
+      sandbox.stub(usecases, 'getOrganizationCampaigns');
+      sandbox.stub(campaignSerializer, 'serialize');
+      codeStub = sandbox.stub();
+      replyStub = sandbox.stub().returns({ code: codeStub });
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+    });
+
+    it('should return the campaigns belonging to the organization', () => {
+      // given
+      const campaigns = factory.buildCampaign();
+      const serializedCampaigns = {
+        data:
+          [
+            {
+              name: 'My awesome campaign',
+              code: 'AZERTY117'
+            }
+          ]
+      };
+      usecases.getOrganizationCampaigns.resolves(campaigns);
+      campaignSerializer.serialize.returns(serializedCampaigns);
+      request = {
+        params: {
+          id: 1
+        }
+      };
+
+      // when
+      const promise = organizationController.getCampaigns(request, replyStub);
+
+      // then
+      return promise.then(() => {
+        expect(usecases.getOrganizationCampaigns).to.have.been.calledOnce;
```

Comment:
J'aurai bien séparé ça en deux `it`
genre `it should call the usecase with the campaign repo` et `it should return the serialized campaign`

Reply:
Plus rajouter les arguments d'appel ?

Reply:
Ok